src/content/docs/errors-inbox/errors-inbox.mdx

```diff
@@ -67,7 +67,33 @@ Error groups are sets of events that make up a unique error.  Error groups are s
 
 Error events get grouped into an error group when they share the same fingerprint.  As events are ingested by New Relic, we run the events through a set of managed rules that output a fingerprint.  Every unique fingerprint has a single error group associated with it.
 
-The New Relic managed rules normalize the error data, identifying and ignoring unique values such as UUIDs, hex values, email addresses, etc. that would cause grouping "like" errors into unique groups. New Relic account ID, entity ID, error class, error message, stack trace, and exception are all data that can impact a fingerprint.
+The New Relic managed rules normalize the error data, identifying and ignoring unique values such as UUIDs, hex values, and email addresses that would cause grouping "like" errors into unique groups. New Relic account ID, entity ID, error class, error message, stack trace, and exception are all data that can impact a fingerprint.
+
+Do any of the following apply to your errors inbox today?
+* Errors are not being grouped in a way that makes the most sense for your application
+* There are multiple groups you would like to see merged
+* Is your inbox “noisy” due to poor grouping and pushing you to exceed the daily limit of unique error groups
+* Does your inbox have an error group that contains unrelated errors?
+
+Having well grouped errors helps errors inbox better triage and diagnose problems in your system. While errors inbox provides a set of default grouping, it is occasionally necessary to make adjustments.
+
+There are two options for adjusting error groups: updating the error message and [using custom attributes](/docs/errors-inbox/errors-inbox/#best-practices) or creating your own error groups.
+
+#### Create your own error groups [#create-error-groups]
+Many of our agents now have a feature that lets your code look at an error and designate an error group before it is submitted to New Relic.  The designated group is then added as the `error.group.name` attribute on the error event.
+
+Many of our agents now have a feature that lets your code look at an error before it is submitted to New Relic and designate the error group you want it to belong to.  The group you designate is then added as the `error.group.name` attribute on the error event. To get started, check out the APM or Browser agent docs on how to specify a custom error group.  This will ensure errors and attributes collected by the agents are preserved and grouped to your liking. 
+
+Supported APM agents:
+
+* [Go](/docs/apm/agents/go-agent/get-started/introduction-new-relic-go/)
+* [Java](/docs/apm/agents/java-agent/getting-started/introduction-new-relic-java/)
+* [Node.js](/docs/apm/agents/nodejs-agent/getting-started/introduction-new-relic-nodejs/)
```

Comment:
@meda123 we don't write the latest releases in docs because it requires a lot of maintenance. I did link the relevant APM pages and people can see the latest supported versions there.